src/enhanced/utils/job/handle-job.ts

```diff
@@ -0,0 +1,67 @@
+import { JobManager, JobHandlerFactory } from '../../../job/jobManager';
```

Comment:
Nothing new here, just moved handle job code from index to a separate file as there are now other functions that this directory exports.